app/uk/gov/hmrc/hmrcemailrenderer/templates/helpers/organisationname/OrgNameHelper.scala

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.hmrcemailrenderer.templates.helpers.organisationname
+
+object OrgNameHelper {
+
+  def capitalised(param: Option[Any]) =
+    param.map { value =>
+      val buffer = scala.collection.mutable.ArrayBuffer.empty[Char]
+      value.toString.copyToBuffer(buffer)
+      buffer.zipWithIndex.map { case (char, idx) =>
+        if (idx == 0 || !buffer(idx - 1).isLetter) char.toUpper
+        else char.toLower
+      }.mkString
+    }
+
+
+  def orgNameOrYou(params: Map[String, Any]): String = {
+    val orgNameParams: List[Option[String]] = List(
+      capitalised(params.getNonEmpty("recipientOrgName"))
+    )
+
+    orgNameParams match {
+      case List(Some(orgName)) => s"$orgName is"
+      case _ => "You are"
```

Comment:
This might be better if it was something along the lines of "case _ => Your Organisation is".

Reply:
_ case can be an individual or sole trader.

Reply:
OrgNameParams has to be more generic for any digital service that wants to create a template that includes an organisation name. My PM doesn't think that having "You are" as a default parameter would be applicable to other templates. We believe that using something along the lines of "The business/company is" would be more appropriate. An alternative to this would be to use $fullname as a parameter for individuals and sole traders.